app/src/main/java/com/example/aurora/MainActivity.java

```diff
@@ -0,0 +1,119 @@
+package com.example.aurora;
+
+import androidx.appcompat.app.AppCompatActivity;
+
+import android.content.Intent;
+import android.os.Bundle;
+import android.view.View;
+import android.widget.ImageView;
+import android.widget.TextView;
+
+public class MainActivity extends AppCompatActivity {
+    //GUI variables on start interface
+
+    private ImageView personalPeacePlayBtn;
+    private ImageView forestThemePlayBtn;
+    private ImageView oceanThemePlayBtn;
+
+    //theme titles
+    private TextView theme1;
+    private TextView theme2;
+    private TextView theme3;
+
+    @Override
+    protected void onCreate(Bundle savedInstanceState) {
```

Comment:
This method is definitely a candidate for refactoring. Try to write short units of code.